components/frontend/src/header_footer/Footer.js

```diff
@@ -149,35 +139,38 @@ function QuoteColumn() {
     ]
     const randomQuote = quotes[Math.floor(Math.random() * quotes.length)]
     return (
-        <FooterCenterColumn>
+        <FooterColumn>
             <FooterItem>
                 <em>{randomQuote[0]}</em>
             </FooterItem>
             <FooterItem>
                 <em>{randomQuote[1]}</em>
             </FooterItem>
             <FooterItem>{randomQuote[2]}</FooterItem>
-        </FooterCenterColumn>
+        </FooterColumn>
     )
 }
 
 export function Footer({ lastUpdate, report }) {
     return (
-        <Segment className="footer" inverted>
-            <Container>
-                <Grid>
-                    <Grid.Row>
-                        <Grid.Column width={1} />
-                        <AboutAppColumn />
-                        {report ? <AboutReportColumn lastUpdate={lastUpdate} report={report} /> : <QuoteColumn />}
-                        <Grid.Column width={1} />
-                        <SupportColumn />
-                    </Grid.Row>
-                </Grid>
-                <Divider inverted section />
-                <Image centered size="mini" src="./favicon.ico" />
+        <Container maxWidth="100%" sx={{ bgcolor: "black", displayPrint: "none", marginTop: "20px", padding: "60px" }}>
```

Comment:
hehe, additional padding and margin

Reply:
Yeah, top margin to create some distance between the subject tables and the footer, and padding so the footer content keeps its distance from the window border.